Fix-User-Guide.md

```diff
@@ -0,0 +1,252 @@
+![logo](https://github.com/culturegraph/metafacture-core/wiki/img/metafacture_small.png)
+
+# Fix User Guide
+
+This document provides an introduction to the Metafacture Fix language (short: Metafix or Fix). Fix is a declarative flow oriented language in which transformations of arbitrary metadata/semi-structured data can be defined using the FIX language. The  Fix language for Metafacture is introduced as an alternative to configuring data transformations with Metamorph. Inspired by Catmandu Fix, Metafix processes metadata not as a continuous data stream but as discrete records. The basic idea is to rebuild constructs from the (Catmandu) Fix language like functions, selectors and binds in Java and combine with additional functionalities from the Metamorph toolbox.
+
+## Part of a metafacture worflow
+Metafacture Fix is a transformation module that can be used in a workflow, for this you have to use this in your pipeline:
+
+Flux-Example:
+```PERL
+   infile
+   | open-file
+   | as-lines
+   | decode-marc21
+   | fix(FLUX_DIR + "fixFile.fix")
+   | encode-json
+   | print
+   ;
+```
+
+- when using the FLUX: 
+- - address the `fix`-module
+- - you can add variables
+- - there are some optiones available
```

Comment:
```suggestion
- - some modules have options available
```

Reply:
This relates to the fix command only, but I deleted this line.